manufacturing-ai-suite/industrial-edge-insights-multimodal/third-party-programs.txt

```diff
@@ -43,78 +50,90 @@ ia-time-series-analytics-microservice:
 
-1. Base docker image: python:3.13.4-slim 
+1. Base docker image: python:3.13.4-slim
 License-Identifier: PSF License
- 
-2. requirements.txt
 
+2. requirements.txt
 License-Identifier: Apache License 2.0
+aiofiles==25.1.0
+----------------------------------------------------------------------
 
-cryptography==44.0.1
-pyOpenSSL==25.1.0
-sortedcontainers==2.4.0
-typing_extensions==4.14.0
-tzdata==2025.2
-
-License-Identifier: MIT License
-aiofiles==24.1.0
-aiosqlite==0.21.0
-cffi==1.17.1
-six==1.17.0
-
-License-Identifier: LGPL-3.0
-asyncua==1.1.5
+License-Identifier: EPL-2.0
+paho-mqtt==1.4.0
+----------------------------------------------------------------------
 
 License-Identifier: BSD License
-numpy==2.3.0
-pandas==2.2.3
-pycparser==2.22
+numpy==2.3.5
+pandas==2.3.1
 python-dateutil==2.9.0.post0
 pytz==2025.2
+six==1.17.0
+tzdata==2025.2
+----------------------------------------------------------------------
 
 License-Identifier: MIT License / Proprietary
-pip==25.1.1
+pip==25.3
 
 ---------------------------------------------------------------------------------------------------------------------------
 
-ia-mqtt-publisher microservice:
+ia-weld-data-simulator:
 ===============================
 
-1. Base docker image: python:3.13.4-slim
+1. Base docker image: python:3.13-slim
 License-Identifier: PSF License
 
+
 2. requirements.txt
+License-Identifier: BSD License
+numpy==2.2.6
+pandas==2.3.2
+----------------------------------------------------------------------
 
 License-Identifier: Apache License 2.0
-cryptography==44.0.1
-pyOpenSSL==25.1.0
-sortedcontainers==2.4.0
-typing_extensions==4.14.0
+opencv-python==4.12.0.88
+paho-mqtt==2.1.0
+----------------------------------------------------------------------
+
+License-Identifier: BSD-like License
+python-dateutil==2.9.0.post0
+pytz==2025.2
 tzdata==2025.2
+----------------------------------------------------------------------
 
 License-Identifier: MIT License
-aiofiles==24.1.0
-aiosqlite==0.21.0
-cffi==1.17.1
 six==1.17.0
+---------------------------------------------------------------------------------------------------------------------------
 
-License-Identifier: LGPL-3.0
-asyncua==1.1.5
+ia-fusion-analytics:
+===============================
 
-License-Identifier: BSD License
-numpy==2.3.0
-pandas==2.2.3
-pycparser==2.22
+1. Base docker image: python:3.13-slim
+License-Identifier: PSF License
+
+2. requirements.txt
+License-Identifier: MIT
+certifi==2025.11.12
+charset-normalizer==3.4.4
+idna==3.11
+msgpack==1.1.2
+numpy==2.3.3
+pandas==2.3.3
 python-dateutil==2.9.0.post0
 pytz==2025.2
+requests==2.32.5
+six==1.17.0
+tzdata==2025.2
+urllib3==2.5.0
+----------------------------------------------------------------------
 
-License-Identifier: MIT License / Proprietary
-pip==25.1.1
-
+License-Identifier: Apache-2.0
+influxdb==5.3.2
+paho-mqtt==2.1.0
 
 ---------------------------------------------------------------------------------------------------------------------------

```

Comment:
Hey @sathyendranv ,
Can you add an entry for the dataset as discussed both here and in the time series folder?